pkg/apis/core/v1beta1/types_seed.go

```diff
@@ -196,6 +196,13 @@ type SeedNetworks struct {
 	Pods string `json:"pods" protobuf:"bytes,2,opt,name=pods"`
 	// Services is the CIDR of the service network. This field is immutable.
 	Services string `json:"services" protobuf:"bytes,3,opt,name=services"`
+	// VPN is the CIDR of the vpn network (spanning vpn-seed-server and vpn-shoot) of shoots on this seed. It must be
+	// disjoint with the Nodes, Pods, and Services networks. It must be a /24 network for IPv4 and /120 for IPv6.
+	// This field is mutable. Changing it leads to a temporary VPN disconnect during the next reconciliation of all shoots
+	// on this seed.
+	// Defaults to "192.168.123.0/24" or "fd8f:6d53:b97a:1::/120" depending on the primary IPFamilies configuration.
+	// +optional
+	VPN *string `json:"vpn,omitempty" protobuf:"bytes,7,opt,name=vpn"`
```

Comment:
FMPOV, it made sense to add the new field after the other network configuration options.

The protobuf generator can handle struct fields in a different order than in the protobuf spec.
When we remove fields, we have to leave tombstones for the protobuf generator, but this applies to the general case, not only here. See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md:

> [...] if the feature is abandoned, or the field name is changed, the field should be removed from the go struct, with a tombstone comment ensuring the field name and protobuf tag are not reused:

There are even k8s core API types that have struct fields that are out-of-order in terms of their protobuf fields: [`PodSpec`](https://github.com/kubernetes/api/blob/039b247b85af57bb7144c894424602b57dd1e98d/core/v1/types.go#L3601), [`Secret`](https://github.com/kubernetes/api/blob/039b247b85af57bb7144c894424602b57dd1e98d/core/v1/types.go#L6970). So we should be safe here.

That being said, I'm willing to change the order if you insist :)

Reply:
Please change the order.

I was not concerned about the protobuf generator at all, but only about humans reading/changing the code. As far as I can tell we append only, which makes this a rare exception that can be easily overlooked. I would like to prevent future (human) errors by simply sticking to the append-only approach.

Reply:
Got it, your reasoning makes sense to me. I moved the new fields to the end of the structs.